examples/hello_world/build.debug.yaml

```diff
@@ -1,12 +0,0 @@
-targets:
```

Comment:
I believe you can still get this with a `debug_options` section in the build.yaml, just fyi

cc @natebosch for confirmation (I think we still need docs for this)

Reply:
You can use something like:

```yaml
targets:
  $default:
    builders:
      build_web_compilers|entrypoint:
        generate_for:
           - web/main.dart
        options:
          compiler: dart2js
          dart2js_args:
            - --fast-startup
            - --trust-type-annotations
            - --trust-primitives
        release_options:
          dart2js_args: # inherits `compiler` from default options, and overrides the args
            - --fast-startup
            - --minify
            - --trust-type-annotations
            - --trust-primitives
```

And if you want an option to only apply in non-release (null value in release) you can use the key `dev_options` which also overrides the defaults from `options` only in non-release builds